ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -549,6 +561,20 @@ objects:
             esac
             echo "Azure region: ${AZURE_REGION}"
 
+            vnetrg=""
+            vnetname=""
+            ctrlsubnet=""
+            computesubnet=""
+            if [[ "${CLUSTER_VARIANT}" =~ "shared-vpc" ]]; then
+              vnetrg="os4-common"
+              case "${AZURE_REGION}" in
+              centralus) vnetname="do-not-delete-shared-vnet";;
```

Comment:
Can we use `do-not-delete-shared-vnet-centralus` and:

```sh
vnetname="do-not-delete-shared-vnet-${AZURE_REGION}"
```
instead of the `case` here?

Reply:
done with commit 85ff0d8